<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Chenzf" name="Command line test" tests="3" failures="1" timestamp="4 Nov 2020 15:00:24 GMT" time="0.849" errors="0">
  <testcase name="testHasMatchUser" time="0.028" classname="com.chenzf.service.UserServiceTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.chenzf.service.UserServiceTest.testHasMatchUser(UserServiceTest.java:39)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- testHasMatchUser -->
  <testcase name="testAddLoginLog" time="0.035" classname="com.chenzf.service.UserServiceTest"/>
  <testcase name="testFindUserByUserName" time="0.057" classname="com.chenzf.service.UserServiceTest"/>
</testsuite> <!-- Command line test -->
